src/include/com/amazonaws/kinesis/video/webrtcclient/Stats.h

```diff
@@ -0,0 +1,321 @@
+    SCTP_TRANSPORT, //!< SCTP transport statistics related to an RTCSctpTransport object
+    TRANSCEIVER, //!< Statistics related to a specific RTCRtpTransceiver
+    RTC_ALL //!< Report all supported stats
+} RTC_STATS_TYPE;
+
+/**
+ * @brief Indicates computing states in the checklist
+ * Reference: https://www.w3.org/TR/webrtc-stats/#rtcstatsicecandidatepairstate-enum
+ */
+typedef enum {
+    FROZEN,    //!< A check for the pair hasn't been performed because it is waiting for other checks to complete
+    WAITING,   //!< A check has not been performed for this pair
+    IN_PROGRESS,//!< A check has been sent for this pair
+    FAILED,    //!< A check for the pair was already done and failed
+    SUCCEEDED  //!< A check for the pair was already done and produced a successful result
+} RTC_STATS_ICE_CANDIDATE_PAIR_STATE;
+
+/**
+ * @brief Set details of the IceAgent based on STUN_ATTRIBUTE_TYPE_USE_CANDIDATE flag
+ * Reference: https://www.w3.org/TR/webrtc/#rtcicerole
+ */
+typedef enum {
+    UNKNOWN,    //!< An agent whose role is undetermined. Initial state
+    CONTROLLING,//!< A controlling agent. The ICE agent that is responsible for selecting the final choice of candidate pairs
+    CONTROLLED  //!< A controlled agent. The iCE agent waits for the controlling agent to select the final choice of candidate pairs
+} RTCIceRole;
+
+/**
+ * @brief DTLS Transport State
+ * Reference: https://www.w3.org/TR/webrtc/#rtcdtlstransport-interface
+ */
+typedef enum {
+    NEW,       //!< DTLS has not started negotiating yet
+    CONNECTING,//!< DTLS is in the process of negotiating a secure connection
+    CONNECTED, //!< DTLS has completed negotiation of a secure connection
+    CLOSED,    //!< The transport has been closed intentionally
+    FAIL       //!< The transport has failed as the result of an error
+} RTCDtlsTransportState;
+
+/**
+ * @brief Defines reasons for quality limitation for sending streams
+ *
+ * Reference: https://www.w3.org/TR/webrtc-stats/#rtcqualitylimitationreason-enum
+ */
+typedef enum {
+    NONE, //!< Resolution as expected. Default value
+    BANDWIDTH, //!< Reason for limitation is congestion cues during bandwidth estimation
+    CPU,//!< The resolution and/or framerate is primarily limited due to CPU load.
+    OTHER, //!< Limitation due to reasons other than above
+} RTCQualityLimitationReason;
+
+/**
+ * @brief Record of duration and quality reason state
+ */
+typedef struct {
+    UINT64 durationInSeconds; //!< Time (seconds) spent in each state
+    RTCQualityLimitationReason qualityLimitationReason; //!< Quality limitation reason
+}qualityLimitationDurationsRecord, PQualityLimitationDurationsRecord;
+
+/**
+ * @brief Record of total number of packets sent per DSCP. Used by RTCOutboundRtpStreamStats
+ * object
+ */
+typedef struct {
+    CHAR dscp[MAX_STRING_LENGTH + 1]; //!< DSCP String
+    UINT64 totalNumberOfPacketsSent; //!< Number of packets sent
+} perDscpPacketsSentRecord, *PperDscpPacketsSentRecord;
```

Comment:
Capitalize the struct names. Perhaps drop the 'per' prefix - something like DscpPacketsSentRecord

Reply:
Done